testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -510,6 +533,12 @@ def test_atom_reordering(self, u, reference_positions):
             assert_almost_equal(atom1.position, atom1_pos, decimal=5)
             assert_almost_equal(atom13.position, atom13_pos, decimal=5)
 
+    def test_additional_colunmns(self, u_add, reference_additional_columns):
+        charges = u_add[0].trajectory[0].data['q'] # this is the universe with just q
+        second  = u_add[1].trajectory[0].data['l'] # this is the universe with both
+        assert_almost_equal(charges, reference_additional_columns['charges'])
```

Comment:
Again compare with the encapuslated `RefLAMMPSDataAdditionalColumns` rather than a local fixture.